### DISCUSSION TITLE
when can we see the submit result and leaderboard ranking?

### DISCUSSION THREAD

**Discussion by khan-yin** [PaddlePaddle/PaddleSpatial#50] [BaiduKDDCup2022] [answered]
Hi， I trained the baseline model and submitted test-1.zip for the competition, but I can not see my score and rank on the leaderboard or my submission record， well, I'm not sure if my submission was successful, could you show me the right way for submission?
![image](https://user-images.githubusercontent.com/50772219/167844925-5bc6a3eb-8a38-4dae-9f05-dc5c0a0fa63e.png)
![image](https://user-images.githubusercontent.com/50772219/167844955-1532c8fc-72d7-4ccb-9ce0-c2ab29ed6890.png)

**Comment by ramber1836** [comment, marked as answer]
The server-side evaluation is ready now.
